pages/index.js

```diff
@@ -1,15 +1,33 @@
-import Head from "next/head";
+import Header from "../components/Header";
 
-export default function Home() {
+export default function About() {
   return (
-    <>
-      <Head>
-        <title>Next.js Starter!</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
```

Comment:
This is an example of what the head should look like. Not visible to the user but still part of the widget tree.